corehq/apps/domain/views.py

```diff
@@ -677,7 +677,8 @@ def can_purchase_credits(self):
     @property
     @memoized
     def plan(self):
-        plan_version, subscription = Subscription.get_subscribed_plan_by_domain(self.domain_object)
+        subscription = Subscription.get_active_subscription_by_domain(self.domain)
+        plan_version = subscription.plan_version
```

Comment:
This should probably check for `subscription` being `None` (there are similar checks in this function)